[0m20:44:39.827250 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA76C9C2F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA795D6E90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA795D7250>]}


============================== 20:44:39.901733 | e57c3ece-aeea-4ca6-8627-9aed1beca931 ==============================
[0m20:44:39.901733 [info ] [MainThread]: Running with dbt=1.7.9
[0m20:44:39.904656 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\LOTUS\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:44:39.907151 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/LOTUS/.dbt')}
[0m20:44:39.908863 [info ] [MainThread]: Creating dbt configuration folder at 
[0m20:45:02.407309 [debug] [MainThread]: Starter project path: C:\Users\LOTUS\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\include\starter_project
[0m20:45:02.503775 [info ] [MainThread]: 
Your new dbt project "quete_etl_elt_DBT" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:45:02.506416 [info ] [MainThread]: Setting up your profile.
[0m20:45:16.134844 [info ] [MainThread]: Profile quete_etl_elt_DBT written to C:\Users\LOTUS\.dbt\profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m20:45:16.137728 [debug] [MainThread]: Command `dbt init` succeeded at 20:45:16.137263 after 36.72 seconds
[0m20:45:16.139015 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA72B96520>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA79B485F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA79796030>]}
[0m20:45:16.140566 [debug] [MainThread]: Flushing usage events
[0m22:15:28.814925 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020A9589C2F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020A981D7ED0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020A981D7B10>]}


============================== 22:15:28.826336 | 5d05822a-ed27-4d3f-937b-9644211566ab ==============================
[0m22:15:28.826336 [info ] [MainThread]: Running with dbt=1.7.9
[0m22:15:28.828587 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\LOTUS\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m22:15:28.830205 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\LOTUS\Documents\Formation_data_analyst\Cours\quetes\ETL-ELT\github\quete_DBT\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m22:15:28.832589 [debug] [MainThread]: Command `dbt run` failed at 22:15:28.832129 after 0.78 seconds
[0m22:15:28.833820 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020A98053490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020A98202E70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020A982585A0>]}
[0m22:15:28.835094 [debug] [MainThread]: Flushing usage events
[0m22:19:19.753444 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017E4619C2F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017E48AD7ED0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017E48AD7B10>]}


============================== 22:19:19.769364 | 7d985b76-9a6d-4c7b-b886-15f74e672c5e ==============================
[0m22:19:19.769364 [info ] [MainThread]: Running with dbt=1.7.9
[0m22:19:19.771950 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\LOTUS\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'D:\\LOTUS\\Documents\\Formation_data_analyst\\Cours\\quetes\\ETL-ELT\\github\\quete_DBT\\logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m22:19:20.014188 [error] [MainThread]: Encountered an error:
Runtime Error
  Credentials in profile "quete_etl_elt_DBT", target "dev" invalid: ['schema'] is not valid under any of the given schemas
[0m22:19:20.017208 [debug] [MainThread]: Command `dbt run` failed at 22:19:20.016740 after 0.98 seconds
[0m22:19:20.018632 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017E48D7DA70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017E48DF0950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017E48D52250>]}
[0m22:19:20.020165 [debug] [MainThread]: Flushing usage events
